modules/migration-mtc-release-notes-1-6.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+
+[id="migration-mtc-release-notes-1-6_{context}"]
+= {mtc-full} 1.6 release notes
+
+The release notes for {mtc-full} ({mtc-short}) version 1.6 describe new features and enhancements, deprecated features, and known issues.
+
+[id="new-features-and-enhancements-1-6_{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+* State migration
++
+You can migrate an application's state by selecting persistent volume claims (PVCs) and Kubernetes objects for migration.
+
+* "New operator version available" notification
++
+The Clusters page of the {mtc-short} web console displays a notification when a new {mtc-full} Operator is available.
+
+[id="deprecated-features-1-6_{context}"]
+== Deprecated features
+
+The following features are deprecated:
+
+* {mtc-short} version 1.4 is no longer supported.
+
+[id="known-issues-1-6_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+* On {product-title} 3.10, the `MigrationController` pod takes too long to restart. The bug report contains a workaround. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986796[*BZ#1986796*])
+
+* Direct volume migration from a classic OpenShift cluster on IBM Cloud fails unless the application is stopped during migration. The IBM block storage plugin does not allow the same volume to be mounted on multiple pods of the same node. As a result, PVCs cannot be mounted on the Rsync pods and on the application pods simultaneously.
```

Comment:
To unblock this, the workload can be quiesced prior to a migration to ensure that there is only ever one pod mounting an RWO volume.